### ISSUE TITLE
Edit an entry?

### ISSUE THREAD

**Issue by Aranad** [notsecure/tox-dns#3]
I've changed my Tox address, so now how do I update my tox-dns entry?

**Comment by GrayHatter** [comment]
do you have access to your old tox address?

**Comment by Aranad** [comment]
Yep my old address was ACCF609BA7578D6AF6B8FB6ECBB926F2938B372CBFBE07460BADC0C359E7145F4C952C27FCCF

**Comment by Aranad** [comment]
(but I can't use it now because I didn't back up my utox data and my drive died - doh!)

**Comment by Aranad** [comment]
You can see that it's my address from the following link which is a historical version of my home page that links to my Github profile and visa-versa:
https://www.organicdesign.co.nz/wiki/index.php?title=Nad/About_me&oldid=120004

**Comment by Aranad** [comment]
aran@utox.org still set to old address, any chance this can be updated soon please?

**Comment by GrayHatter** [comment]
Without access to the private key there's no way to make this happen securely. You can open this issue at https://github.com/notsecure/tox-dns but again, bottom of my todo list because theres' no way for me to know you've ever had that toxid

**Comment by whoizit** [comment]
same for me, I need to change atmxz@utox.org, I have all profiles, what I should do?

**Comment by whoizit** [comment]
this should use login:pass pair for able to change tox id.

**Comment by whoizit** [comment]
I fix it, no need for me now, but u should do it able changable with login:pass pair
